cli.js

```diff
@@ -12,7 +13,23 @@ const cli = cac('create-nuxt-app')
 cli
   .command('[out-dir] [options]', 'Generate in a custom directory or current directory')
   .option('--edge', 'To install `nuxt-edge` instead of `nuxt`')
-  .action((outDir = '.') => {
+  .option('--info', 'Print out debugging information relating to the local environment')
+  .action(async (outDir = '.') => {
+    const hasInfoArg = process.argv.slice(2)[0] !== '--edge'
```

Comment:
```suggestion
    const hasInfoArg = process.argv.slice(2)[0] !== '--edge'
```

Could we make the check more stable? :thinking: 	
Something like "if `--info` is included as an argument at all"?

Reply:
`cac` takes care of any other arguments passed in and warns appropriately except `--edge` and `--info`.

Reply:
Maybe I'm getting this wrong but:

In case there will be a third flag (say `--foo`) added in the future, people would have to touch that line of code again as they'd have to check against `--edge` and `foo`. So I'd probably explicitly check for `=== '--info'` :thinking:

Reply:
Gotcha :+1: